### ISSUE TITLE
Can not acess the index.html when installing in a headless vps.

### ISSUE THREAD

**Issue by tangdou1** [yacy/yacy_search_server#276]

**Comment by gangarer** [comment]
By default Yacy only allows access from localhost. You can create a ssh-tunnel to get around this to set up an administrator_account and allow access from anywhere. Also you can set the needed options either in yacy.conf after installation or in yacy.init before.

**Comment by gangarer** [comment]
The options you are looking for are
adminAccountUserName=  
adminAccountForLocalhost=  #should be set to false
adminAccountBase64MD5=  #can be set via <yacy-home>/bin/passwd.sh <new-password>.

**Comment by gangarer** [comment]
Sorry, i didnt read your question right. Admin access is set to localhost by default, but you should be able to reach the index.html. What exactly is your setup? Do you have any firewall enabled at your vps?

**Comment by tangdou1** [comment]
When I follow these steps in 
https://github.com/yacy/yacy_search_server/wiki/Run-Yacy-in-a-VPS
I can not aceess yacy through http://my vps's ip:8090.

**Comment by gangarer** [comment]
No, they miss one step. After step 5. you need to compile yacy. You only downloaded the source code. As step 6. run `ant clean all` in the yacy directory. Maybe you also need to install ant if it isnt installed already. After that you should be able to start yacy with './startYACY.sh'.
